test/spec/features/plugin/uninstall_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe 'plugin uninstall' do
+  before(:each) do
+    run('kontena plugin install aws')
+  end
+
+  it 'removes installed plugin' do
+    k = run('kontena plugin uninstall --force aws')
+    expect(k.code).to eq(0)
```

Comment:
Should we also verfiy that we get at least some of the known plugins? (DO, AWS, vagrant, ...)